README.developers.old

```diff
@@ -0,0 +1,83 @@
+Building and running on Vivid Desktop (15.04)
```

Comment:
We don't need this, if anyone is interested in it, they can use the git history.